src/main/labview/labview.ts

```diff
@@ -463,7 +465,7 @@ export class LabView implements IDisposable {
           });
         }
 
-        async function jlabDesktop_setUIMode(uiMode) {
+        async function jlabDesktop_setUIMode(uiMode, showTOCInZenMode) {
```

Comment:
I would make the second argument an object with options like `options: {showTOCInZenMode?: boolean}` to make it easy to add/remove arguments in the future, but this is not really a public API (I think?) so probably does not matter much.